src/sphinx_book_theme/assets/styles/content/_margin.scss

```diff
@@ -38,6 +38,47 @@ $content-fullwidth-width: percentage(100% / $content-max-width);
   }
 }
 
+label.margin-toggle {
+  margin-bottom: 0em;
+  &.marginnote-label {
+    display: none;
+  }
+  @media (max-width: $breakpoint-md) {
+    cursor: pointer;
+    &.marginnote-label {
+      display: inline;
+      &:after {
+        content: "\2295";
+      }
+    }
+  }
+}
+
+input.margin-toggle {
+  display: none;
+  @media (max-width: $breakpoint-md) {
+    &:checked + .sidenote,
+    &:checked + .marginnote {
+      display: block;
+      float: left;
+      left: 1rem;
+      clear: both;
+      width: 95%;
+      margin: 1rem 2.5%;
+      position: relative;
+    }
+  }
+}
+
+span.sidenote,
+span.marginnote {
```

Comment:
Note that we should add these to the list of classes the intersectionobserver is watching:

https://github.com/executablebooks/sphinx-book-theme/blob/227e1995652a3d09aec426d13104a6e430696bd2/src/sphinx_book_theme/assets/scripts/index.js#L133-L141